integrations/cassandra/src/cassandra.go

```diff
@@ -23,48 +25,59 @@ type argumentList struct {
 
 const (
 	integrationName    = "com.newrelic.cassandra"
-	integrationVersion = "1.1.0"
+	integrationVersion = "2.0.0-beta"
 )
 
 var (
 	args argumentList
 )
 
 func main() {
-	integration, err := sdk.NewIntegration(integrationName, integrationVersion, &args)
-	fatalIfErr(err)
-	log.SetupLogging(args.Verbose)
+	i, err := integration.New(integrationName, integrationVersion, integration.Args(&args))
+	if err != nil {
+		panic(err)
+	}
 
-	fatalIfErr(jmx.Open(args.Hostname, strconv.Itoa(args.Port), args.Username, args.Password))
+	e := i.LocalEntity()
+	l := i.Logger()
+
+	fatalIfErr(l, jmx.Open(args.Hostname, strconv.Itoa(args.Port), args.Username, args.Password))
 	defer jmx.Close()
 
-	if args.All || args.Metrics {
+	if args.All() || args.Metrics {
 		rawMetrics, allColumnFamilies, err := getMetrics()
-		fatalIfErr(err)
+		fatalIfErr(l, err)
 
-		ms := integration.NewMetricSet("CassandraSample")
+		s, err := e.NewMetricSet("CassandraSample", metricSetAttributes...)
```

Comment:
these `metricSetAttributes` are metrics that are present in `columnFamilyMetrics` map and we don't use that one to populate the `CassandraSample` metric set. Does it make sense to use it here?